sound/soc/sof/intel/hda-dsp.c

```diff
@@ -624,7 +624,7 @@ static int hda_suspend(struct snd_sof_dev *sdev, bool runtime_suspend)
 #endif
 
 	/* power down DSP */
-	ret = hda_dsp_core_reset_power_down(sdev, chip->host_managed_cores_mask);
```

Comment:
@bardliao can we please add the ASOC: SOF: Intel: hda prefix in the commit title?

Reply:
> @bardliao can we please add the ASOC: SOF: Intel: hda prefix in the commit title?

Done